civihr_default_theme/scss/partials/_menu.scss

```diff
@@ -62,3 +62,98 @@
         }
     }
 }
+
+.chr_header__settings-menu {
+  position: relative;
+  display: inline-block;
+  width: 50px;
+  height: 50px;
+  text-align: center;
+  float: right;
+
+  &:hover {
+    background-color: $crm_gray-darker-alt;
+
+    > .chr_header__settings-sub-menu {
+      display: block;
+    }
+  }
+
+  .chr_header__settings-sub-menu {
+    background: $crm_gray-darker-alt;
+    display: none;
+    position: absolute;
+    right: 0;
+    z-index: 99;
+
+    li {
+      text-align: left;
+    }
+
+    a {
+      color: $light-pale;
+      font-size: 1.3rem;
+      display: block;
+      padding: 0 30px 0 20px;
+      white-space: nowrap;
+
+      &:hover {
+        background: $crm_brand-secondary;
+        color: $light;
+      }
+
+      > i {
+        margin-right: 10px;
+      }
+    }
+  }
+}
+
+.chr_header__home-menu {
+
+  .chr_header__home-sub-menu {
```

Comment:
Sorry if I dont understand this properly. BE talks about having unique name for elements to 
1. Avoid long names
2. Having the ability the move elements without worrying about changing the name of classes.

But in this case,
1. We avoided long names by having unique name.
2. But as sub menu is never going to be outside menu, I think we should have it nested inside the menu.

Let me know your views on this.